frontend/src/store/actionTypes.js

```diff
@@ -0,0 +1 @@
+export const CATEGORIES = "CATEGORIES";
```

Comment:
try to use semantic names like `GET_ALL_CATEGORIES` or `SAVE_CATEGORIES`

Reply:
done